<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColdHotTransferDialog</class>
 <widget class="QDialog" name="ColdHotTransferDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>580</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QWidget" name="containerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>400</width>
      <height>420</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <widget class="QToolButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>350</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>发起资产变更</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>币种</string>
     </property>
    </widget>
    <widget class="QComboBox" name="accountComboBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>发起账户</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QToolButton" name="okBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>350</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>备注</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="memoLineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="assetComboBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>转出地址</string>
     </property>
    </widget>
    <widget class="QLabel" name="fromLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>280</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
    <widget class="QLabel" name="toLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>280</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>转入地址</string>
     </property>
    </widget>
    <widget class="QComboBox" name="typeComboBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <item>
      <property name="text">
       <string>冷钱包转热钱包</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>热钱包转冷钱包</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="timeLineEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>260</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>86400</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>240</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>终止确认时间（秒）</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="amountLineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>数量</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
